### PR TITLE
Update Syphon runtime to 46

### PR DESCRIPTION
**Pull request by ghost**
- Update Syphon runtime to 46 since runtime version 44 has reached end-of-life.
- Fix appdata papercuts

### CODE REVIEW

**Comment by ghost** [comment]
@0x1a8510f2,

I've updated Syphon's runtime to 46 since runtime version 44 has reached end-of-life.

The update fails due to a missing screenshot. (https://matrix.org/docs/projects/images/syphon.png) Could you provide a URL to replace it?

Thanks.

File: org.syphon.Syphon.metainfo.xml [hunk 1]
```diff
@@ -16,13 +17,9 @@
   <developer_name>Syphon Organization, LLC</developer_name>
   <description>
     <p>Syphon aims to be built on the foundations of privacy, branding, and user experience in an effort to pull others away from proprietary chat clients to the matrix protocol.</p>
-
-    <p>Website: https://syphon.org</p>
-    <p>Source Code: https://github.com/syphon-org/syphon</p>
-    <p>Matrix Space: https://matrix.to/#/#syphon-space:matrix.org</p>
```
Comment (0x1a8510f2) [inline org.syphon.Syphon.metainfo.xml:22]: I would like to keep some kind of reference to the Matrix space. Most likely still in the description.

**Review by 0x1a8510f2 — COMMENTED** [review]
Thanks very much for this! As long as you've confirmed that the app runs as it should, I'm happy to merge.

Only thing is I'd like to keep a reference somewhere, to the Matrix space.

**Comment by 0x1a8510f2** [comment]
As for the screenshot, feel free to use the ones in the original repo's README: https://github.com/syphon-org/syphon/blob/dev/readme.md

File: org.syphon.Syphon.metainfo.xml (hunk 1 above)
Comment (ghost) [reply org.syphon.Syphon.metainfo.xml:22]: It's not possible to add plain URLs to the descriptions, so I removed https part.

**Comment by ghost** [comment]
single sing on button does not work.

resim

**Pull request closed without merging**